build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   
   <PropertyGroup>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

Comment:
This seems to be a breaking change in the pack targets. We're passing a nuspec file and before we didn't have to have this set. Now, it tries to call targets we don't have (this is not a csproj) without this.